src/Authorization/Services/Implementation/ContextHandler.cs

```diff
@@ -230,6 +234,13 @@ protected XacmlResourceAttributes GetResourceAttributeValues(XacmlContextAttribu
                 if (attribute.AttributeId.OriginalString.Equals(XacmlRequestAttribute.InstanceAttribute))
                 {
                     resourceAttributes.InstanceValue = attribute.AttributeValues.First().Value;
+                    string[] instanceValues = resourceAttributes.InstanceValue.Split('/');
+                    resourceAttributes.ResourceInstanceValue = instanceValues[1];
+                }
+
+                if (attribute.AttributeId.OriginalString.Equals(XacmlRequestAttribute.ResourceRegistryInstanceAttribute))
```

Comment:
Burde dette være en else if for å unngå dobbelinnsending av instanceid og ressursinstanceid?

Reply:
Det samme kan man si om duplikater av samme attributt og org/app og resourceId.

Har absolutt lyst til å legge på mer input validering på hva vi godtar på requests, men føler det bør gjøres i en egen endring.

Reply:
Oppgir man både instanceId og resourceInstanceId på er det samme som med duplikater av alle andre attributter, at det er siste er vel det som blir gjeldende